dev/tasks/tasks.yml

```diff
@@ -1833,12 +1833,32 @@ tasks:
         HDFS: 2.9.2
       run: conda-python-hdfs
 
-  test-conda-python-3.7-spark-master:
+  test-conda-python-3.6-spark-2.4.6:
     ci: github
     template: docker-tests/github.linux.yml
     params:
       env:
-        PYTHON: 3.7
+        PYTHON: 3.6
+        SPARK: "v2.4.6"
+      # use the master branch of spark, so prevent reusing any layers
+      run: --no-leaf-cache --env ARROW_PRE_0_15_IPC_FORMAT=1 conda-python-spark
+
+  test-conda-python-3.7-spark-branch-3.0:
```

Comment:
looks like this should be named python-3.8 also

Reply:
Fixing it.